appengine_config.py

```diff
@@ -90,20 +90,21 @@ def save(self):
     sys.path.insert(0, pil_path)
 
 THIRD_PARTY_LIBS = [
-    os.path.join(ROOT_PATH, 'third_party', 'backports.functools_lru_cache-1.5'),
-    os.path.join(ROOT_PATH, 'third_party', 'beautifulsoup4-4.7.1'),
-    os.path.join(ROOT_PATH, 'third_party', 'bleach-3.1.0'),
+    os.path.join(ROOT_PATH, 'third_party', 'backports.functools_lru_cache-1.6.1'),
+    os.path.join(ROOT_PATH, 'third_party', 'beautifulsoup4-4.9.0'),
+    os.path.join(ROOT_PATH, 'third_party', 'bleach-3.1.5'),
     os.path.join(ROOT_PATH, 'third_party', 'callbacks-0.3.0'),
     os.path.join(ROOT_PATH, 'third_party', 'future-0.17.1'),
     os.path.join(ROOT_PATH, 'third_party', 'gae-cloud-storage-1.9.22.1'),
     os.path.join(ROOT_PATH, 'third_party', 'gae-mapreduce-1.9.22.0'),
     os.path.join(ROOT_PATH, 'third_party', 'gae-pipeline-1.9.22.1'),
     os.path.join(ROOT_PATH, 'third_party', 'graphy-1.0.0'),
     os.path.join(ROOT_PATH, 'third_party', 'html5lib-python-1.0.1'),
-    os.path.join(ROOT_PATH, 'third_party', 'mutagen-1.42.0'),
-    os.path.join(ROOT_PATH, 'third_party', 'simplejson-3.16.0'),
+    os.path.join(ROOT_PATH, 'third_party', 'packaging-20.3'),
```

Comment:
is this a new lib (why?)? (Also, add this in new sorted order)

Reply:
"the `packaging` library is added so that `bleach` works correctly"